docs/_ui.js

```diff
@@ -15,6 +15,10 @@ export const LiveEditor = props => {
     case '!':
       return <LivePreview mdx={lang === '!mdx'} code={code} />
     default:
-      return <Pre p={3} mt={4} mb={4} bg="gray" children={props.children} />
+      return (
+        <Pre p={3} mt={4} mb={4} bg="gray">
+          {props.children}
```

Comment:
If you do not want it, you can avoid `react/no-children-prop: 0` by adding it to eslint.
* [react/no-children-prop](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-children-prop.md)